.github/CODEOWNERS

```diff
@@ -543,7 +543,7 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 /sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/ @pakrym @JoshLove-msft @jsquire
 
 # ServiceLabel: %Service Bus %Service Attention
-/sdk/servicebus/Microsoft.Azure.ServiceBus/                    @axisc
+/sdk/servicebus/Microsoft.Azure.ServiceBus/                    @shankarsama @DorothySun216 
```

Comment:
Unfortunately this change will not split of the test notifications because that is based on the yml file path. If you want those to be unique you would need to add another entry with a full path to the yml file like: 

```
/sdk/servicebus/tests.data.yml       @shankarsama @DorothySun216 
```

Reply:
@jsquire confirmed that the notifications are indeed split for Event Hubs where they used this model and not the full path

https://github.com/Azure/azure-sdk-for-net/blob/b33567b3e3bdbefcd5cbb4b01e4c179b3b81da3b/.github/CODEOWNERS#L294-L307

Reply:
The code owners are probably split up correctly but the pipeline notifications I don't believe are split up as you expect.

Track 1 (i.e. tests.data.yml) DevOps notification group:
[net - servicebus - tests.data](https://dev.azure.com/azure-sdk/internal/_settings/teams?subjectDescriptor=vssgp.Uy0xLTktMTU1MTM3NDI0NS03MjEyMjY4NDEtNDc1NTgxMjYxLTI3MDQxNzIzOTItMzQ2Mzk5NTc2NS0xLTE4MDIwMDgzMjEtMTg1NTE0OTEyNy0yOTY4Nzg0MzYwLTI1ODI1NzE3Nw)

vs

Track 2 (i.e. tests.yml) DevOps notification group:
[net - servicebus - tests](https://dev.azure.com/azure-sdk/internal/_settings/teams?subjectDescriptor=vssgp.Uy0xLTktMTU1MTM3NDI0NS03MjEyMjY4NDEtNDc1NTgxMjYxLTI3MDQxNzIzOTItMzQ2Mzk5NTc2NS0xLTM5OTEyOTUwOTYtMzYwNDgyMzcxLTMxNTQzMDU0MzktMzYxNzM3OTc1Nw)

They both only contain Jesse and Josh. If you want the other owners for the different pipelines you will have to fully specify the path to the yml file given they are in the same root directory.

Reply:
Thanks @weshaggard! Created https://github.com/Azure/azure-sdk-for-net/pull/23663

Reply:
Interesting; I haven't been receiving notifications for Event Hubs T1 tests since we split them, but have continued to receive the T2 notifications.    Does this mean that nobody is receiving the T1 notifications, I wonder?

Reply:
That's curious.   I haven't been seeing them and confirmed they're not going to Junk or being caught by rules.